src/AcceptanceTestsShared/When_using_outbox_with_transaction_scope.cs

```diff
@@ -9,47 +9,42 @@
 [TestFixture]
 public class When_using_outbox_with_transaction_scope : NServiceBusAcceptanceTest
 {
-    [Test]
-    public async Task Should_float_transaction_scope_into_handler()
+    [TestCase(IsolationLevel.ReadCommitted)]
+    [TestCase(IsolationLevel.Serializable)]
```

Comment:
ReadCommited and Snapshot are the isolation levels that we allow that all databases can handle